### PR TITLE
ka - makes unhealthy cows cheaper to sell

### LINKED ISSUE #15: FEATURE: When cows are sold, the selling prices is adjusted based on cow health
#  User Story

* As a player
* I can sell cows only for a price that reflects the cow health
* so that the game is more accurate, fair, interesting.

### Discussion

Currently, the cow prices is the price at which cows are both bought and sold.
Instead, we want cows to be sold at a fraction of the cow price based on cow health.

Formula: `Selling Price = Cow Price * Cow Health`

### Acceptance Criteria

- [x] When I sell a cow, the selling price is the `Cow Price * Cow Health`

### Implementation Todos

- [x] Adjust the cow selling logic in the appropriate controller
- [x] Check that the frontend logic still works properly

### PR DESCRIPTION
**Pull request by kirrarista**
Decreases sell value of cows with health below 100% for players. New profit calculation is performed by multiplying the market sell value by average cow health. 

To test: 
1. go to https://proj-happycows-kirrarista.dokku-05.cs.ucsb.edu/
2. buy some cows
3. sell come cows
4. go to Admin->Manage Jobs-> Set Cow Health For A Specific Commons
5. change cow health
6. go to your commons
7. sell some cows
8. notice the change in profits per cow sold

Closes #15

### CODE REVIEW

**Comment by cyrus895** [comment]
LGTM

File: frontend/package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "env-cmd": "^10.1.0",
     "eslint": "^8.16.0",
     "eslint-plugin-react": "^7.30.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
```
Comment (GDW1) [inline frontend/package.json:47]:
```suggestion
```

**Review by GDW1 — DISMISSED** [review]
Just one edit on the imports. Make sure that package lock doesnt have changes

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**